etc/helm/examples/microsoft-values.yaml

```diff
@@ -5,6 +5,7 @@ deployTarget: MICROSOFT
 
 pachd:
   storage:
+    storageURL: az://pachyderm-test
```

Comment:
This can be more accurate if it's azblob://foo

Reply:
Good catch, thanks.